src/explore-education-statistics-frontend/src/modules/table-tool/components/TimePeriodDataTable.tsx

```diff
@@ -46,6 +46,20 @@ const TimePeriodDataTable = (props: Props) => {
       [options => options.length],
     );
 
+    // strips siblingless "Total" filters (columns/rows)
+    for (let i = 0; i < sortedFilters.length; i += 1) {
+      const sortedFilter = sortedFilters[i];
+      if (sortedFilter.length === 1) {
+        if (typeof sortedFilter[0].label === 'string') {
+          const { label } = sortedFilter[0];
+          if (label.toUpperCase() === 'TOTAL') {
```

Comment:
I really don't like this as the reason for some of Ben's API changes was to add the `totalValue` property which describes the value that we're interested in.

We should actually add an additional property to `CategoryFilter` like `isTotal` and assign this when it is constructed. We can then easily extract this information here.